src/utils/dates.ts

```diff
@@ -0,0 +1,9 @@
+export const daysInMonth = (
+  year: number,
+  month: number,
+): number => new Date(year, month + 1, 0).getDate();
+
+export const emptyDaysInMonth: (
+  year: number,
+  month: number,
+) => number = (year, month) => new Date(year, month, 1).getDay();
```

Comment:
I'm curious about your decision here. The `daysInMonth` function above is nearly identical, but is defined in a way to me that is very clear, and exactly how I would choose to define it. This one has the inline type def which make it very confusing to me. It took me way too long to understand what I was looking at here, lol. I don't want to jump to any conclusions about why you did it like this, so I would love to hear your thought process!

Reply:
Yeah, I agree and find inline typing to be visually more confusing. At Codecademy, we were encouraged to use an inline type definition for utility functions. So, I'd be happy to switch it over!